src/components/content/order/services/Services.tsx

```diff
@@ -105,6 +105,9 @@ function Services(): React.JSX.Element {
                                                 <span className={serviceOrderStyles.serviceTypeOptionDescription}>
                                                     {item.content}
                                                 </span>
+                                                <span className={serviceOrderStyles.serviceTypeOptionVendor}>
```

Comment:
better to move the namespace to another custom component and use it everywhere directly.

Reply:
fixed